### DISCUSSION TITLE
Demodulize class name on mailer method lookup

### DISCUSSION THREAD

**Discussion by StanisLove** [excid3/noticed#331] [General]
Hi, mailer method is defined as an underscored class name if the `method` option wasn't provided

https://github.com/excid3/noticed/blob/df561f348cc33283e4cc62f711f7d9a47b811a12/lib/noticed/delivery_methods/email.rb#L36-L43

But if I put my notification class to a module the result string will contain the name of that module as a prefix.

```ruby
Employees::NotifyAboutNewOrder.name.underscore #=> "employees/notify_about_new_order"
```

I think it is appropriate to pre-demodulize a class name here.

**Comment by excid3** [comment]
Makes sense to me. Do you want to submit a PR for that?

**Comment by aka-nez** [comment]
@excid3 hi! What you think about @StanisLove PR?
